packages/management/tkg/bundle/config/packageinstalls/service-account.yaml

```diff
@@ -5,14 +5,14 @@ metadata:
   name: tanzu-tkg-package-sa
```

Comment:
can you also rename this ServiceAccount to better convey which package installation uses this SA?
something I've come up for tanzu-framework-manager https://github.com/vmware-tanzu/tanzu-framework/pull/1848/files#diff-b0f4231cf94dd415b922ab3391584ef24dcbe3ebd251907d4853c07bad2a9a6fR5